android/src/main/java/io/cobrowse/reactnative/CobrowseIOModule.java

```diff
@@ -399,4 +235,13 @@ public void accessibilityServiceIsRunning(final Promise promise) {
     promise.resolve(CobrowseAccessibilityService.isRunning(getReactApplicationContext()));
   }
 
+  @ReactMethod
+  public void overwriteSessionIndicator() {
+    CobrowseIODelegatesWithSessionControl delegatesWithSessionControls = (CobrowseIODelegatesWithSessionControl) Proxy.newProxyInstance(
+      CobrowseIOCommonDelegates.class.getClassLoader(),
+      new Class[] { CobrowseIODelegatesWithSessionControl.class },
```

Comment:
Can we use the interface names directly here rather than creating an intermediate class? The Internet made it seem that way but I haven't tried it and stackoverflow can't always be trusted 😅

Reply:
Not sure I follow according to the [docs](https://docs.oracle.com/javase/7/docs/api/java/lang/reflect/Proxy.html#newProxyInstance(java.lang.ClassLoader,%20java.lang.Class[],%20java.lang.reflect.InvocationHandler)) the second argument to newProxyInstance is an array of classes which I think is what we create here. I tried to just use `CobrowseIODelegatesWithSessionControl.class` but that didn't work (android studio tells me wrap it in a class array). Maybe i'm misunderstanding your comment?

Reply:
NIce. I had misunderstood your initial comment, sorry. Updated